benchmark/sparse/linear_regression/keras_tf_model.py

```diff
@@ -0,0 +1,47 @@
+"""
+Linear Regression model with sparse synthetic data for Keras
+"""
+
+import time
+
+import numpy as np
+from keras.layers import Dense, Input
+from keras.models import Sequential
+from keras.optimizers import SGD
+from scipy.sparse import csc_matrix
+
+# Fix random seed for reproducibility
+np.random.seed(7)
```

Comment:
Can we create a common file to run the benchmark and it will create a random dataset instead of fixed seed? Some thing like `run_sparse_benchmark.py`, and the logic will be:
```
1. Generate random dataset
2. Pass the dataset to `keras_tf_model.py` to train and it returns acc, loss, time to train
3. Pass the dataset to `mxnet_sparse_model.py` to train and it returns acc, loss, time to train
```

In this way you can make sure same data is passed and each time you run it, data will be random generated

Reply:
Done